packages/launchdarkly-react-client-sdk/src/withLDProvider.tsx

```diff
@@ -69,14 +71,16 @@ export function withLDProvider(config: ProviderConfig) {
           ldClient: undefined,
         };
 
-        const { options } = config;
-        if (options) {
-          const { bootstrap } = options;
-          if (bootstrap && bootstrap !== 'localStorage') {
-            this.state = {
-              flags: camelCaseKeys(bootstrap),
-              ldClient: undefined,
-            };
+        if (typeof config !== 'function') {
```

Comment:
To be honest, I'm not familiar enough with the `localStorage` option to see how this will work with this new logic.